iFixit/react-commerce
# Troubleshooting: Link primary author's name to their profile

We want to link the primary author to their profile and the "contributors" message (if there is one) to the history page.

## QA Notes
Please confirm that it matches the description in the issue.
https://react-commerce-git-troubleshooting-use-separate-l-020767-ifixit.vercel.app/Vulcan/Dryer_Not_Spinning should have multiple contributors; note that the links will point to the Vercel page, not the iFixit page, so they won't work directly when clicked. You'll need to edit the URL you end up on to switch from the Vercel hostname to the `cominor.com` hostname.

Closes ifixit/ifixit#47194
